src/modules/logicModule/LM_PC_Staking_v1.sol

```diff
@@ -262,9 +267,11 @@ contract LM_PC_Staking_v1 is
             return rewardValue;
         }
 
-        return (_getRewardDistributionTimestamp() - lastUpdate) // Get the time difference between the last time it was updated and now (or in case the reward period ended the rewardEnd timestamp)
```

Comment:
I explicitly structured the comments here in a way to explain this complex interaction in a easier way.
You can set brackets here, but please keep it in the original structure